src/esp/physics/PhysicsManager.cpp

```diff
@@ -304,9 +304,9 @@ int PhysicsManager::addArticulatedObjectInstance(
   existingArticulatedObjects_.at(aObjID)->setSceneInstanceAttr(
       aObjInstAttributes);
 
-  // set articulated object's user-defined attributes, if any exist in scene
+  // merge articulated object's user-defined attributes, if any exist in scene
   // instance.
-  existingArticulatedObjects_.at(aObjID)->setUserAttributes(
+  existingArticulatedObjects_.at(aObjID)->mergeUserAttributes(
```

Comment:
Nit: calling at(aObjId) is not free. The reference should really be cached between these calls, don't you think?

Reply:
Every little bit helps, yeah